src/main/java/org/erp/school/client/Client.java

```diff
@@ -1,21 +1,18 @@
-package org.erp.school.model;
+package org.erp.school.client;
 
 import lombok.Data;
+import org.erp.school.client.child.activity.Activity;
+import org.erp.school.client.child.address.Address;
+import org.erp.school.client.child.user.User;
+import org.erp.school.client.child.contract.Contract;
+import org.erp.school.client.child.document.Document;
+import org.erp.school.client.child.request.ClientRequest;
 import org.hibernate.annotations.GenericGenerator;
 
-import javax.persistence.CascadeType;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.FetchType;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.JoinTable;
-import javax.persistence.OneToMany;
-import javax.persistence.OneToOne;
-import javax.persistence.Table;
+import javax.persistence.*;
```

Comment:
we need to check what is your configuration here. Are you using IntelliJ or Eclipse? Did you change the defaults values?

Reply:
Intellij, I will check that out, I could have accidentally put star when merging.

Reply:
Yes, it seems Intelij actually doing that